Sources/Misc/Deprecations.swift

```diff
@@ -130,7 +130,10 @@ public extension Purchases {
     /**
      * Enable automatic collection of Apple Search Ads attribution. Defaults to `false`.
      */
-    @available(*, deprecated, message: "Use Purchases.shared.attribution.enableAdServicesAttributionTokenCollection() instead")
+    @available(*, deprecated, message: """
+    Use Purchases.shared.attribution.enableAdServicesAttributionTokenCollection() instead.
+    AdClient doesn't work after February 7, 2023  so this boolean doesn't have any effect.
+    """)
     @objc static var automaticAppleSearchAdsAttributionCollection: Bool = false
```

Comment:
looks good, we should probably entirely obsolete for v5 since it doesn't work right now (cc @fire-at-will)

Reply:
addressed in #3994